rsmtool/utils/prmse.py

```diff
@@ -0,0 +1,328 @@
+"""
+Utility classes and functions related to computing test
+theory based evaluations.
+
+The derivations and formulas were provided by Matt Johnson.
+
+:author: Anastassia Loukina (aloukina@ets.org)
+:author: Nitin Madnani (nmadnani@ets.org)
+
+:organization: ETS
+"""
+
+import pandas as pd
+import numpy as np
+
+
+def get_n_human_scores(human_scores):
+    """
+    Compute number of human scores for each response
+
+    Parameters
+    ----------
+    human_scores : array-like of shape (n_samples, n_ratings)
+        Human ratings for each response
+
+    Returns
+    -------
+    n_scores : array-like of shape (n-samples)
+        Total number of not None human scores
+    """
+    n_scores = (~np.isnan(human_scores)).sum(axis=1)
+    return n_scores
+
+
+def variance_of_errors(human_scores):
+    '''Compute variance of human errors.
+
+    Parameters
+    ----------
+    human_scores : array-like of shape (n_samples, n_ratings)
+        Human ratings for each response
+
+    Returns
+    -------
+    variance_of_errors : float
+        Estimated variance of human errors
+    '''
+
+    # if the user passsed pandas Dataframe,
+    # we convert it to an array since
+    # in these computations having column names
+    # interferes with the computation
```

Comment:
```suggestion
    # if the user passed a pandas data frame, convert it to a numpy array since
    #  having column names interferes with the following computations
```